ui/app/components/dashboard/overview.ts

```diff
@@ -30,13 +28,13 @@ export default class OverviewComponent extends Component<Args> {
   // for HVD clusters, this is the `admin` namespace
   get shouldShowClientCount() {
     const { version, isRootNamespace } = this.args;
-    const { flags, namespace } = this;
+    const { namespace } = this;
 
     // don't show client count if this isn't an enterprise cluster
     if (!version.isEnterprise) return false;
 
     // HVD clusters
-    if (flags.isHvdManaged && namespace.currentNamespace === 'admin') return true;
```

Comment:
`currentNamespace` could mean the full namespace path is `ns1/admin` and the user is in `admin`. we want to differentiate between namespaces called `admin/` and the actual administrative, top-level namespace which is what this PR addresses